system_tests/twonodeslong_test.go

```diff
@@ -141,10 +141,10 @@ func testTwoNodesLong(t *testing.T, dasModeStr string) {
 			if err != nil {
 				Fail(t, err)
 			}
-		}
-		_, err := EnsureTxSucceeded(ctx, l1client, tx)
-		if err != nil {
-			Fail(t, err)
+			_, err = EnsureTxSucceeded(ctx, l1client, tx)
```

Comment:
EnsureTxSucceeded was done outside the loop on purpose. That way we only we for a transaction one time and not 30 times. Probably deserves a comment.

Reply:
I moved it inside the loop since it started failing with timeouts if it was outside the loop.